### DISCUSSION TITLE
Analyze Model feature Bug

### DISCUSSION THREAD

**Discussion by minasihene** [sql-bi/Bravo#493] [General]
I am using the analyze feature to identify columns that are not used in my model. But I realize that the scroll goes back to the top within a few seconds when I  am not scrolling or when I take my mouse cursor from the screen

**Comment by albertospelta** [comment]
@minasihene Unfortunately we are not able to reproduce, does this issue happen consistently with all models? 
Can you provide us the steps you perform after starting Bravo to reproduce this behavior?

**Reply by minasihene** [reply]
Hi Alberto,
This happens when I open Bravo from the external tools tab in power BI desktop
for all my reports that are connected to PBI Online Dataset.

**Reply by albertospelta** [reply]
Does this happen even with local reports, not connected to a PBI dataset ?
